tests/test_app/TestApp/Controller/Component/SomethingWithCookieComponent.php

```diff
@@ -17,6 +17,8 @@
 
 /**
  * SomethingWithCookieComponent class
+ *
+ * @property Component\CookieComponent $Cookie
```

Comment:
I think that is incorrect from the namespace
also should be FQCN but no big deal here